region/maridia/outer.json

```diff
@@ -1183,667 +1022,388 @@
+          "to": [
+            {
+              "id": 2,
+              "requires": [ "Morph" ],
+              "unlock": null
+            }
+          ]
+        },
+        {
+          "from": 7,
+          "to": [
+            {
+              "id": 2,
+              "requires": null,
+              "unlock": null
+            },
+            {
+              "id": 3,
+              "requires": null,
+              "unlock": null
+            },
+            {
+              "id": 8,
               "requires": [
                 {"or": [
                   "canSwim",
                   {"and": [
                     "canSuitlessMaridia",
                     {"or": [
-                      {"and": [
-                        "canCrabClimb",
-                        "Super"
-                      ]},
-                      {"and": [
-                        "HiJump",
-                        "canSpringBallJump"
-                      ]},
-                      {"and": [
-                        "Ice",
-                        "Grapple"
-                      ]},
-                      {"and": [
-                        "Ice",
-                        "HiJump"
-                      ]}
+                      "HiJump",
+                      "canSpringBallJumpMidAir",
+                      "canUseFrozenEnemies",
+                      "Grapple"
                     ]}
                   ]}
                 ]}
               ],
               "unlock": null
             },
             {
-              "id": 3,
+              "id": 10,
               "requires": [
-                {"or": [
-                  "canSwim",
-                  {"and": [
-                    "canSuitlessMaridia",
-                    {"or": [
-                      "Ice",
-                      {"and": [
-                        "HiJump",
-                        "canSpringBallJump"
-                      ]}
-                    ]}
-                  ]}
-                ]}
-              ],
-              "unlock": null
-            },
-            {
-              "id": 4,
-              "requires": [
-                {"or": [
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "canCrabClimb",
-                    "Super"
+                {"or":[
+                  "canGravityJump",
+                  {"and":[
+                    "canSwim",
+                    "HiJump",
+                    "canInsaneWalljump"
                   ]},
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "Grapple",
-                    {"or": [
-                      "Ice",
-                      {"and": [
-                        "HiJump",
-                        "canSpringBallJump"
-                      ]}
-                    ]}
+                  {"and":[
+                    "canSwim",
+                    "HiJump",
+                    "SpeedBooster",
+                    "canWalljump"
```

Comment:
This feels like a TrickyWalljump

Reply:
trickyWallJumps are usually delayed wallJumps or the like. Getting the correct speed to get the correct jump height does appear to be tricky, but you're supposed to get high enough to just do a simple walljump off the edge. What's your though on replacing that SpeedBooster by a canTrickyDashJump?

Reply:
Sounds good